bin/katello-certs-check

```diff
@@ -112,6 +112,24 @@ function check-ca-bundle () {
     fi
 }
 
+function check-cert-encoding () {
+    printf "Checking server certificate's encoding: "
+    openssl x509 -inform per -in $CERT_FILE -noout -text &> /dev/null
```

Comment:
shouldn't this be `-inform pem` instead of `per`? That's at least what https://wiki.openssl.org/index.php/Manual:X509(1) suggests.